ansible/roles/nomad-client/templates/40-client.hcl

```diff
@@ -33,6 +33,14 @@ client {
     read_only = {{volume.read_only|bool|lower}}
   }
 {% endfor %}
+
+{% if nomad_meta %}
```

Comment:
This causes an `AnsibleUndefinedVariable` error.  Please change this line to `{% if nomad_meta|default(false) %}`.